lib/rentlinx/default.rb

```diff
@@ -6,9 +6,11 @@ module Default
     USER_AGENT = "Rentlinx Ruby Client #{Rentlinx::VERSION}".freeze
 
     class << self
-      def headers
+      def headers(token: nil)
```

Comment:
In my opinion the defaults should always return the same values. Augmenting them should occur after the defaults have been retrieved. See how octokit uses them: 

https://github.com/octokit/octokit.rb/blob/master/lib/octokit/default.rb